sale_configurator_option/models/account_move_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.depends("sale_line_ids")
```

Comment:
I think we should add the concept of parent_id/child_ids here. And fill this field when we create the invoice